server/core/include/irods/miscServerFunct.hpp

```diff
@@ -142,11 +142,13 @@ irods::error readStartupPack(
 
 irods::error setRECacheSaltFromEnv();
 
+#if 0
 irods::error get_script_output_single_line(
     const std::string&              script_language,
     const std::string&              script_name,
     const std::vector<std::string>& args,
     std::string&                    output );
+#endif
```

Comment:
Do we still want to keep this?

This function was used to run python scripts from an admin-approved directory (e.g. get_agent_pid.py, etc).

iRODS 5 doesn't use this function anymore, but it's not clear if we'll need it at a later time. I don't suspect we will since we want to avoid execution of external scripts.

Thoughts?

Reply:
Removed this function since we can always restore it or build a new version of it using the git history.